src/pages/index.tsx

```diff
@@ -3,21 +3,26 @@ import * as _ from "lodash"
 import * as React from "react"
 import Layout from "../components/Layout"
 import SEO from "../components/Seo"
+import { string } from "prop-types"
 
 interface PostNode {
   node: {
     excerpt: string
     frontmatter: {
       date: string
       title: string
-      author: string
+      author: SingleAuthor[]
     }
     fields: {
       slug: string
     }
   }
 }
 
+interface SingleAuthor {
+  singleAuthor: string
+}
+
```

Comment:
Personally I would just call this `Author` since `Author[]` implies it is a list of authors. Similarly I just call the interface above `Post` and leave the `Node` off the ending.